packages/special-pages/pages/duckplayer/src/js/index.js

```diff
@@ -301,18 +302,20 @@ const Comms = {
      * Validates that the input string is a valid video id.
      * If so, returns the video id otherwise returns false.
      * @param {string} input
-     * @returns {(string|false)}
+     * @returns {(string|null)}
      */
     validateVideoId: (input) => {
-        if (/^[a-zA-Z0-9-_]+$/g.test(input)) {
-            return input
+        if (typeof input !== 'string') return null
+        const subject = input.slice(0, 11)
+        if (/^[a-zA-Z0-9-_]+$/g.test(subject)) {
+            return subject
```

Comment:
Should we return the 11nth onwards chars if they're valid? Do we know for certain that 11 is enough always etc?

Reply:
In this instance, we never want anything after the first 11 (other params are handled elsewhere) and yeah that number seems stable.